.github/workflows/build-tag.yml

```diff
@@ -11,9 +11,11 @@ jobs:
     steps:
     - name: Checkout code
       uses: actions/checkout@v3
-
+    # TODO: In the future, with-node should be an optional param in the action.
```

Comment:
Note, composer should also be parameterized, as we will want a tag action either way, but no need to commit `vendor` if there are no non-dev dependencies.